src/styles/index.css

```diff
@@ -202,7 +202,7 @@ html .ProseMirror {
 }
 
 /* Titres */
-html body .note-editor-container .akEditor p,
+html body .note-editor-container .ak-editor-content-area p,
```

Comment:
This isn't great but the banner will be broken otherwise. I don't think it's a problem though unless we intended to modify the styles of the editor toolbar paragraphs?